numpy/linalg/umath_linalg.c.src

```diff
@@ -1306,29 +1342,30 @@ init_@lapack_func@(EIGH_PARAMS_t* params, char JOBZ, char UPLO,
 
     return 0;
 }
+/**end repeat**/
+
 
+/**begin repeat
```

Comment:
Actually, these are like this because they're unmodified, and git has decided to produce a diff to suggest they have been. The actual change was just to move init_@lapack_func@ after call_@lapack_func@, but one of these functions looks very much like another

Do you want me to add a STY commit adding spacing on top of everything else?